apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html

```diff
@@ -918,3 +925,8 @@ <h1>{{baseCurrencySymbol}}{{ totalDue | giddhCurrency }}<span *ngIf="baseCurrenc
     </div>
 </ng-template>
 
+<!-- Bulk export modal -->
+<ng-template #BulkExport>
```

Comment:
Do use camelCase for template variable names than PascalCase
```suggestion
<ng-template #bulkExport>
```